wp-web-push/wp-web-push-admin.php

```diff
@@ -74,11 +52,10 @@ public function options() {
         wp_die(__('Invalid value for `Registration Behavior`', 'wpwebpush'));
       }
 
-      $triggers_option = array_key_exists('webpush_triggers', $_POST) ? $_POST['webpush_triggers'] : array();
-      $allowed_trigger_values = $this->array_values_recursive($ALLOWED_TRIGGERS);
-      foreach ($triggers_option as $trigger_option) {
-        if (!in_array($trigger_option, $allowed_trigger_values)) {
-          wp_die(sprintf(__('Invalid value in Push Triggers: %s', 'wpwebpush'), $trigger_option));
+      $triggers_option = $_POST['webpush_triggers'];
```

Comment:
Yes. To reproduce the warnings, open the settings page, make sure "New Post" is not selected.

I see the following notice and warnings:
- Notice: Undefined index: webpush_triggers in /var/www/html/wordpress/wp-content/plugins/wp-web-push/wp-web-push-admin.php on line 55
-  Warning: Invalid argument supplied for foreach() in /var/www/html/wordpress/wp-content/plugins/wp-web-push/wp-web-push-admin.php on line 56
- Warning: in_array() expects parameter 2 to be array, null given in /var/www/html/wordpress/wp-content/plugins/wp-web-push/wp-web-push-admin.php on line 112